spec/exp_descriptor_fr_spec.rb

```diff
@@ -375,7 +375,7 @@ def desc(expression, opts = {})
       end
 
       it 'year increments' do
-        expect(desc('0 0 0 1 MAR * 2010/5')).to eq('À 0:00 AM, le 1 de chaque mois, seulement en mars, tous les 5 ans, commence en 2010')
+        expect(desc('0 0 0 1 MAR * 2010/5')).to eq('À 12:00 AM, le 1 de chaque mois, seulement en mars, tous les 5 ans, commence en 2010')
```

Comment:
I'm not sure what to do about that since Ruby's strftime seems to follow the same standard as [ANSI C's](https://www.systutorials.com/docs/linux/man/3-strftime/):
```
%p
Either "AM" or "PM" according to the given time value, or the corresponding strings for the
current locale. Noon is treated as "PM" and midnight as "AM". (Calculated from tm_hour.)
```

Reply:
Actually the relevant options are:
```
%H
The hour as a decimal number using a 24-hour clock (range 00 to 23). 
(Calculated from tm_hour.)
%I
The hour as a decimal number using a 12-hour clock (range 01 to 12). 
(Calculated from tm_hour.)
%k
The hour (24-hour clock) as a decimal number (range 0 to 23); single digits are preceded by a blank. (See also %H.) 
(Calculated from tm_hour.) (TZ)
%l
The hour (12-hour clock) as a decimal number (range 1 to 12); single digits are preceded by a blank. (See also %I.) 
(Calculated from tm_hour.) (TZ)
```
If users prefer `0:00 AM` they can use the 24-hour clock (which has an option in `cronex` but appears to be non-functioning).